SPECS/coreutils/coreutils.spec

```diff
@@ -1,35 +1,39 @@
 Summary:        Basic system utilities
 Name:           coreutils
 Version:        8.30
-Release:        5%{?dist}
+Release:        6%{?dist}
 License:        GPLv3
-URL:            http://www.gnu.org/software/coreutils
-Group:          System Environment/Base
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://www.gnu.org/software/coreutils
 Source0:        http://ftp.gnu.org/gnu/coreutils/%{name}-%{version}.tar.xz
 # make this package to own serial console profile since it utilizes stty tool
 Source1:        serial-console.sh
 Patch0:         http://www.linuxfromscratch.org/patches/downloads/coreutils/coreutils-8.30-i18n-1.patch
 Patch1:         http://www.linuxfromscratch.org/patches/downloads/coreutils/coreutils-8.10-uname-1.patch
+# Upstream community agreed to not fix this
+Patch2:         CVE-2016-2781.nopatch
+# Fixed in coreutils-8.30-i18n-1.patch
```

Comment:
Is it an issue with our tooling or NIST's database? If it's the latter, have they got an e-mail asking to update their info?

Reply:
It's neither an issue with tooling or NIST. The security vulnerability still exists. The fix would causes regressions (see this [link](https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.28/v2.28-ReleaseNotes)), so the upstream community decided to not patch it.